src/main/resources/schema/type.graphqls

```diff
@@ -1,6 +1,141 @@
+type Vote{
+  id: ID!
+  targetID: ID!
+  voters: [User]
+  voteFor: Int
+  voteAgainst: Int
+}
+
+type Comment{
+  id: ID!
+  userID:ID!
+  message: String
+  votes: [Vote]
+  replies: [Reply]
+  createdAt: String
+  updatedAt: String
+}
+
+type Reply {
```

Comment:
I don't think we should differentiate replies from comments - replies are simply comments to comments, they will be stored in the database as some hierarchical tree structure with no differentiation.